### PR TITLE
Fix zebrad logs

### PR DESCRIPTION
**Pull request by oxarbitrage**
Revert https://github.com/ZcashFoundation/zebra/pull/840 and configure as specified at https://github.com/ZcashFoundation/zebra/pull/840#issuecomment-670136282

### CODE REVIEW

File: zebrad/src/application.rs [hunk 1]
```diff
@@ -111,18 +111,13 @@ impl Application for ZebradApp {
             .as_ref()
             .expect("config is loaded before register_components");
 
+        let default_filter = if command.verbose { "debug" } else { "info" };
```
Comment (teor2345) [inline zebrad/src/application.rs:114]: Have you tested if the non-server commands output any info-level logs?
If they do, using info as a default will break revhex and generate output, and be annoying for help, version, and no arguments.

Let's make sure that the revhex output is exact, and the output of the other commands starts with the expected first line.

**Review by teor2345 — CHANGES_REQUESTED** [review]
Thanks for this fix!

If we want to use info as the default log level for non-server commands, we need to make sure those commands don't generate any info-level logs by default. So let's make the acceptance tests more strict.

File: zebrad/src/application.rs (hunk 1 above)
Comment (oxarbitrage) [reply zebrad/src/application.rs:114]: Thanks, i just checked that. No info level log is printed in `generate`, `help`, `revhex` and `version` commands.

**Review by dconnolly — APPROVED** [review]
lgtm!

File: zebrad/src/application.rs (hunk 1 above)
Comment (teor2345) [reply zebrad/src/application.rs:114]: I opened #845 and #847 so we test for accidental logs in command output.

**Review by teor2345 — APPROVED** [review]

**Pull request merged**